From gerard.lledo@gmail.com  Tue Jun  2 14:42:15 2009
From: Gerard Lledo <gerard.lledo@gmail.com>
Date: Wed, 27 May 2009 17:35:19 +0300
Subject: Staging: comedi: cb_pcidas.c: Fix build warning (type mismatch)
Cc: Gerard Lledo <gerard.lledo@gmail.com>
Message-ID: <1243434919-10192-3-git-send-email-gerard.lledo@gmail.com>


spin_lock_irqsave expects flags to be unsigned long, not unsigned int.

Signed-off-by: Gerard Lledo <gerard.lledo@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/cb_pcidas.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/comedi/drivers/cb_pcidas.c
+++ b/drivers/staging/comedi/drivers/cb_pcidas.c
@@ -1593,7 +1593,7 @@ static void handle_ao_interrupt(struct c
 	struct comedi_cmd *cmd = &async->cmd;
 	unsigned int half_fifo = thisboard->fifo_size / 2;
 	unsigned int num_points;
-	unsigned int flags;
+	unsigned long flags;
 
 	async->events = 0;
 
